source/content/sftp.md

```diff
@@ -1,41 +0,0 @@
----
-title: On-Server Development
-description: Develop on Pantheon directly via SFTP.
-contributors:  [scottmassey]
-permalink:  docs/:basename
-tags: [files, sftp, code]
-searchboost: 50
-layout: video
-type: video
-contenttype: [doc]
-innav: [true]
-categories: [sftp]
-cms: [--]
-audience: [development]
-product: [--]
-integration: [--]
----
-
-<Youtube src="ktesh9SiHfc" title="On Server Development" />
```

Comment:
@stevector this video does not exist anywhere else in the docs. Do we want to preserve it somewhere like we did for other videos from the 2016 "Self-Serve Training" initiative

Reply:
I'm ok with losing it from the Docs site. I'm currently working with pre-sales on some updated demo videos.